### ISSUE TITLE
Package Request: modern-unix

### ISSUE THREAD

**Issue by Cryptiiiic** [ProcursusTeam/Procursus#857]
https://github.com/ibraheemdev/modern-unix

**Comment by m1stadev** [comment]
Most of the packages listed there are already added, we'd just need to add `ag`, `bottom`, `broot`, `cheat`, `choose`, `curlie`, `delta`, `dog`, `dust`, `duf`, `fd`, `glances`, `gping`, `gtop`, `httpie`, `hyperfine`, `lsd`, `mcfly`, `procs`, `sd`, `tldr`, `xh`, and `zoxide`.

**Comment by TheRealKeto** [comment]
> Some of the packages listed there are already added, we'd just need to add `ag`, `bottom`, `broot`, `cheat`, `choose`, `curlie`, `delta`, `dog`, `dust`, `duf`, `fd`, `glances`, `gping`, `gtop`, `httpie`, `hyperfine`, `lsd`, `mcfly`, `procs`, `sd`, `tldr`, `xh`, and `zoxide`.

Let's use ``python3-psutil`` instead of ``bottom``, since a package with that name (that is unrelated to the project) is already in Procursus.

**Comment by m1stadev** [comment]
Alternatively, the `bottom` package currently on Procursus could be renamed to bottomify, as that's the name of the binary that's installed with that package.

**Comment by sunflsks** [comment]
> Some of the packages listed there are already added, we'd just need to add `ag`, `bottom`, `broot`, `cheat`, `choose`, `curlie`, `delta`, `dog`, `dust`, `duf`, `fd`, `glances`, `gping`, `gtop`, `httpie`, `hyperfine`, `lsd`, `mcfly`, `procs`, `sd`, `tldr`, `xh`, and `zoxide`.

We already have `fd`, so that can be taken off the list as well

**Comment by TheRealKeto** [comment]
> Alternatively, the `bottom` package currently on Procursus could be renamed to `bottomify`, as that's the name of the binary that's installed with that package.

Well, in reality, packages included in that README are alternatives to alternatives, and these packages have almost the same **basic** function; I don't see a reason to have ``gtop`` and ``glances`` on the same system, since these packages provide the same basic function.

**Comment by CRKatri** [comment]
> Well, in reality, packages included in that README are alternatives to alternatives, and these packages have almost the same **basic** function; I don't see a reason to have ``gtop`` and ``glances`` on the same system, since these packages provide the same basic function.

Using that logic, what's the point of having htop when we have top, and what's that point of top when we have ps.

**Comment by TheRealKeto** [comment]
> > Well, in reality, packages included in that README are alternatives to alternatives, and these packages have almost the same **basic** function; I don't see a reason to have `gtop` and `glances` on the same system, since these packages provide the same basic function.
> 
> Using that logic, what's the point of having htop when we have top, and what's that point of top when we have ps.

Those have noticeable changes which set them apart entirely; nowadays, these "alternatives" have almost the same features, just a different codebase; I'll likely end up renaming the package unrelated to ``modern-unix`` to something more different, just give me time before deciding on the naming of it.

**Comment by nyuszika7h** [comment]
Why not just make them separate packages? `modern-unix` seems to be just a non-authorative collection made by some random person, the whole repo is just a README. Then that makes the concern about similar functionality moot, because realistically nobody is going to install the entirety of Procursus on their devices, but it's good to have alternatives.

**Comment by m1stadev** [comment]
I don't believe the intent of this request was to have all of these utilities bundled into one package, but to have all of them added as separate packages to Procursus, for those who preferred them over alternatives.

**Comment by CRKatri** [comment]
> realistically nobody is going to install the entirety of Procursus on their devices

Try me.

**Comment by nyuszika7h** [comment]
<img src="https://cdn.discordapp.com/emojis/792934328236638267.png?v=1" width="48" />

**Comment by asdfugil** [comment]
<img src="https://cdn.discordapp.com/emojis/792934328236638267.png?v=1" alt="youtube.com/dQw4w9WgXcQ" size="48"></img>

**Comment by Doregon** [comment]
<img src="https://cdn.discordapp.com/emojis/792934328236638267.png?v=1" alt="youtube.com/dQw4w9WgXcQ" size="48"></img>
